dspace-server-webapp/src/test/java/org/dspace/app/rest/ClarinItemImportControllerIT.java

```diff
@@ -470,4 +472,53 @@ public void testImportAuthorityAndConfidenceInMetadata() throws Exception {
         assertEquals(dcRelationValue.getAuthority(), String.valueOf(AUTHORITY));
         assertEquals(dcRelationValue.getConfidence(), CONFIDENCE);
     }
+
+    @Test
+    public void importItemsMappedCollections() throws Exception {
+        context.turnOffAuthorisationSystem();
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and two collections.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                .withName("Parent Community")
+                .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                .withName("Sub Community")
+                .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+        Collection col3 = CollectionBuilder.createCollection(context, child1).withName("Collection 3").build();
+
+        //2. Public item that is readable by Anonymous with different subjects
+        Item publicItem1 = ItemBuilder.createItem(context, col1)
+                .withTitle("Public item 1")
+                .withIssueDate("2017-10-17")
+                .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                .withSubject("ExtraEntry")
+                .build();
+        context.restoreAuthSystemState();
+
+        // It is owning collection
+        String col1SelfLink = "https://localhost:8080/spring-rest/api/core/collections/" + col1.getID();
+        String col2SelfLink = "https://localhost:8080/spring-rest/api/core/collections/" + col2.getID();
+        String col3SelfLink = "https://localhost:8080/spring-rest/api/core/collections/" + col3.getID();
+        List<String> collectionSelfLinksList = new ArrayList<>();
+        collectionSelfLinksList.add(col1SelfLink);
+        collectionSelfLinksList.add(col2SelfLink);
+        collectionSelfLinksList.add(col3SelfLink);
+
+        ObjectMapper mapper = new ObjectMapper();
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(post("/api/clarin/import/item/" +
+                                publicItem1.getID() + "/mappedCollections")
+                                .content(mapper.writeValueAsBytes(collectionSelfLinksList))
+                                .contentType(org.springframework.http.MediaType.APPLICATION_JSON))
+                        .andExpect(status().isOk());
+
+        Item updatedItem = itemService.find(context, publicItem1.getID());
+        assertEquals(updatedItem.getCollections().size(), 3);
```

Comment:
maybe get collections from updatedItem as var

Reply:
It is not necessary